test/e2e/container_probe.go

```diff
@@ -43,7 +43,7 @@ var _ = Describe("Probing container", func() {
 		expectNoError(err)
 		startTime := time.Now()
 
-		Expect(wait.Poll(poll, 90*time.Second, func() (bool, error) {
+		Expect(wait.Poll(poll, 120*time.Second, func() (bool, error) {
```

Comment:
I divided the test into two phases, and each of them is given 120s (120s for running, 120s for ready).

Note that the second test "expects" the test to time out when waiting for readiness, so I only set 90s for it.